web-app/js/portal/prototypes/NonCollapsingAccordionLayout.js

```diff
@@ -30,7 +29,16 @@ Ext.ux.NonCollapsingAccordionLayout = Ext.extend( Ext.layout.Accordion, {
     beforeExpandPanel : function(panel) {
         var panelToCollapse = this.currentlyExpandedPanel;  // A holder for the previously selected panel
         this.currentlyExpandedPanel = panel;                // Set the new panel as the currently expanded one
-        panelToCollapse.collapse();                         // Collapse the previously selected panel
+
+        if (panelToCollapse.collapsed) {
+            // fixes #1823
+            setTimeout(function() {
+                panelToCollapse.collapse();
+            }, 400 );
+        }
+        else {
+            panelToCollapse.collapse();                     // Collapse the previously selected panel
```

Comment:
@dnahodil Those comments are a hangover from the original author. I wont mind if all of the comments disappear, or are in line with our code base, as we now have made big changes to that original code. maybe the "fixes #1823" is still useful?